### PR TITLE
SpacesOffCommand

### PR DESCRIPTION
**Pull request by Pho3niX90**
To turn of spaces, and remove the need to remove spaces with code.

### CODE REVIEW

File: src/main/java/com/github/pires/obd/commands/protocol/SpacesOffCommand.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package com.github.pires.obd.commands.protocol;
+
+import android.os.Parcel;
```
Comment (pires) [inline src/main/java/com/github/pires/obd/commands/protocol/SpacesOffCommand.java]: This is a pure Java library. I can' accept Android code imports. Also, it seems it's not even needed.

File: src/test/java/com/github/pires/obd/commands/SpacesOffCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package com.github.pires.obd.commands.protocol;
+
+import junit.framework.TestCase;
+
+/**
+ * Created by sjanvan1 on 2015/09/15.
```
Comment (pires) [inline src/test/java/com/github/pires/obd/commands/SpacesOffCommandTest.java]: Remove this comment.

File: src/test/java/com/github/pires/obd/commands/SpacesOffCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+ */
+public class SpacesOffCommandTest extends TestCase {
```
Comment (pires) [inline src/test/java/com/github/pires/obd/commands/SpacesOffCommandTest.java]: This doesn't follow the structure of this project unit-tests.

See https://github.com/pires/obd-java-api/blob/master/src/test/java/com/github/pires/obd/commands/RPMCommandTest.java

**Comment by Pho3niX90** [comment]
Should be fixed now. Missed the import when clearing out a test I did earlier.

**Comment by pires** [comment]
See https://github.com/pires/obd-java-api/blob/master/src/test/java/com/github/pires/obd/commands/RPMCommandTest.java

**Comment by pires** [comment]
@Pho3niX90 will you add tests?

**Comment by Pho3niX90** [comment]
@Pires Sure. Have completely forgot about this. Will add tests during the day

**Comment by Pho3niX90** [comment]
@pires okay I was browsing through the tests folder to get a "template", however I see there are no tests for any of the AT commands? I suppose this could be because the do not return any useful data, only "OK"?

**Pull request closed without merging**